lib/recipe/widgets/recipe_probes_screen.dart

```diff
@@ -0,0 +1,242 @@
+                                      .read<RecipeBloc>()
+                                      .add(ProbeOptionSelected(
+                                          recipe: state.recipes[recipeIndex],
+                                          probeIndex: index,
+                                          answer: answer!)),
+                                  selectedItem: (state.recipes[recipeIndex]
+                                                  .probes[index].answer ==
+                                              null ||
+                                          state.recipes[recipeIndex]
+                                                  .probes[index].answer ==
+                                              '')
+                                      ? state.recipes[recipeIndex].probes[index]
+                                          .optionsList()[0]
+                                      : state.recipes[recipeIndex].probes[index].answer),
+                            )
+                          ],
+                        ),
+                      )),
+                    );
+                  }),
+            ),
+          ),
+        ],
+      );
+    });
+  }
+}
+
+class ProbesPrompt extends StatelessWidget {
+  final int recipeIndex;
+  final String? assignedFoodItemId;
+  const ProbesPrompt(
+      {Key? key, required this.recipeIndex, this.assignedFoodItemId})
+      : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return BlocBuilder<RecipeBloc, RecipeState>(builder: (context, state) {
+      if (state.recipes[recipeIndex].recipeType != 'Standard Recipe') {
+        return const ListTile(
+          title: Text('Probes are only a feature of Standard recipes'),
+          subtitle: Text('Use a standard recipe to add probes'),
+          tileColor: Colors.blue,
+        );
+      } else if (assignedFoodItemId != null &&
+          state.recipes[recipeIndex].probesChecked &&
+          state.recipes[recipeIndex].probesStandard &&
+          state.recipes[recipeIndex].recipeType == 'Standard Recipe' &&
+          state.recipes[recipeIndex].probes.isNotEmpty) {
+        return const ListTile(
+          title: Text('This is a standard recipe.'),
+          subtitle: Text('Confirm recipe volume on Recipe Details page'),
+          tileColor: Colors.green,
+        );
+      } else if (assignedFoodItemId != null &&
+          state.recipes[recipeIndex].probesChecked &&
+          !state.recipes[recipeIndex].probesStandard &&
+          state.recipes[recipeIndex].recipeType == 'Standard Recipe' &&
+          state.recipes[recipeIndex].probes.isNotEmpty) {
+        return const ListTile(
+          title: Text('This is a modified recipe.'),
+          subtitle: Text('Add or remove ingredients on the ingredients page'),
+          tileColor: Colors.red,
+        );
+      } else {
+        return const SizedBox
+            .shrink(); // Empty widget used in official Material codebase
+      }
```

Comment:
There is quite a lot of logic in this UI screen file, which would ideally be in the BLoC, but I think since there hopefully won't be any more additions to this, we can leave it as it is 😊.